traitlets/config/tests/test_configurable.py

```diff
@@ -368,7 +368,7 @@ class SomeSingleton(SingletonConfigurable):
             pass
 
         class DefaultConfigurable(Configurable):
-            a = Integer(config=True)
+            a = Integer().tag(config=True)
             def _config_default(self):
                 if SomeSingleton.initialized():
                     return SomeSingleton.instance().config
```

Comment:
Just one or two of the simplest existing tests. All I really care about is keeping track of `a = Trait(config=True)` still working and loading its value from the config object.